sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamTest.scala

```diff
@@ -813,6 +813,7 @@ trait StreamTest extends QueryTest with SharedSparkSession with TimeLimits with
         case (key, None) => sparkSession.conf.unset(key)
       }
       sparkSession.streams.removeListener(listener)
+      StateStore.stop()
```

Comment:
Shall we leave a code comment for the reason we put this here? We already have StateStore.stop() in afterEach in various test suites, and future reviewer would like to understand why we can't simply put StateStore.stop() in afterEach. (I get that, just wanted to help future reviewers.)